config/jest/setupTests.ts

```diff
@@ -1,6 +1,9 @@
 // jest-dom adds custom jest matchers for asserting on DOM nodes.
 import "@testing-library/jest-dom/extend-expect";
 
+process.env.REACT_APP_EVERGREEN_URL = "test-evergreen.com";
+process.env.REACT_APP_GRAPHQL_URL = "test-graphql.com";
```

Comment:
oh it's because these things are `undefined` in the Jest environment which causes my `auth.test.tsx` tests to fail 😔 So I just set them to a dummy value